src/test/shared/sam/cli/samCliConfiguration.test.ts

```diff
@@ -85,20 +85,6 @@ describe('SamCliConfiguration', () => {
         assert.strictEqual(timesCalled, 1)
     })
 
-    it('location provider detects a file', async () => {
-        const fakeCliLocation = path.join(tempFolder, 'fakeSamCli')
-
-        const samCliConfig: SamCliConfiguration = new DefaultSamCliConfiguration(settingsConfiguration, {
-            getLocation: async (): Promise<string | undefined> => {
-                return Promise.resolve(fakeCliLocation)
-            },
-        })
-
-        await samCliConfig.initialize()
-
-        assert.strictEqual(samCliConfig.getSamCliLocation(), fakeCliLocation)
```

Comment:
this test checks nothing useful, and fails now because initalize() expects a workspace.